gadget/ondisk.go

```diff
@@ -124,9 +120,6 @@ type OnDiskStructure struct {
 
 	// Node identifies the device node of the block device.
 	Node string
-	// CreatedDuringInstall is true when the structure has properties indicating
-	// it was created based on the gadget description during installation.
-	CreatedDuringInstall bool
```

Comment:
Removal of this field should help the OnDisk and LaidOut structs started in PR #9106 .